homeassistant/components/calendar/__init__.py

```diff
@@ -133,11 +152,16 @@ def state_attributes(self):
         event = normalize_event(event)
         return {
             "message": event["message"],
+            "status": event["status"],
```

Comment:
You're making changes to the entity model. All changes to the entity model need to be approved in the architecture repository first as per https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model